objectivec/cxx_api.h

```diff
@@ -10,17 +10,30 @@
 #pragma clang diagnostic ignored "-Wdocumentation"
 #endif  // defined(__clang__)
 
+// paths are different when building the Swift Package Manager package as the headers come from the iOS pod archive
```

Comment:
[binary target](https://developer.apple.com/documentation/packagedescription/target/binarytarget(name:path:)) have very limited options.

I think if we would have `module.modulemap` then this could work. However, I have no idea how to combine the two. binary target and module map. 

[This](https://www.hackingwithswift.com/articles/87/how-to-wrap-a-c-library-in-swift) might help? not sure

Reply:
Don't have time currently but would be nice if we can. I tried setting headerSearchPath for the objectivec target but that only affects the build of that and doesn't help. I assume any module map would need to be in the ORT xcframework for it to work.